frappe/public/js/frappe/form/layout.js

```diff
@@ -144,8 +144,16 @@ frappe.ui.form.Layout = class Layout {
 				fieldname: "__details",
 			};
 			let first_tab = this.fields[1].fieldtype === "Tab Break" ? this.fields[1] : null;
+
 			if (!first_tab) {
-				this.fields.splice(1, 0, default_tab);
+				this.fields.splice(0, 0, default_tab);
+			} else {
+				// reshuffle __newname field to accomodate under 1st Tab Break
+				let newname_field = this.fields.find((df) => df.fieldname === "__newname");
+				if (newname_field.get_status(this) === "Write") {
```

Comment:
need to handle case where field wasn't found?

Reply:
- `get_doctype_fields` is not called if the `fields` object is already set (for dialogs, etc.)
- In that case, you will notice an error like this in your code:

![image](https://user-images.githubusercontent.com/16315650/195532243-f89cb4e3-cada-4dec-b1cf-fc4a978945bd.png)

Reply:
But do we use tab breaks in Dialogs?

Reply:
Fixed!